packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -399,6 +400,47 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
   void _endOfTestVerifications() {
     verifyTickersWereDisposed('at the end of the test');
   }
+
+  static TestTextInput _testTextInput;
+  static EditableTextState _focusedEditable;
```

Comment:
The `static` here doesn't quite seem right.  Maybe these should be on the TestWidgetsFlutterBinding ?